install.sh

```diff
@@ -248,6 +248,9 @@ install_deb () {
       yes="--n"
       yesapt="-y"
   fi
+  echo "Updating package indexes..."
+  ${SUDO} apt-get update -y
```

Comment:
I don't think `apt-get update` needs `-y` - at least, I've never seen a prompt from it

Reply:
updated to remove `-y`